src/roaring.c

```diff
@@ -129,6 +129,29 @@ void roaring_bitmap_add_many(roaring_bitmap_t *r, size_t n_args,
     }
 }
 
+void roaring_bitmap_contains_multi(roaring_bitmap_t *r, size_t n_args,
+                             const uint32_t *vals, bool *results) {
+    if (n_args == 0) return;
+    memset(results, 0, n_args);
```

Comment:
`sizeof(bool)` is not guaranteed to be `1`, so this may not clean as many results as we expect.
Besides, it may be best to just specify that `results` should be `0`-initialized and avoid duplicating computation when it is.

Reply:
agreed

Reply:
At a minimum, I think we should go with...

```C++
for(...) { results[k] = false; }
```

if we are going to initialize at all. It does not seem warranted to use `memset`.